### PR TITLE
cnf network: fix rdma flake

### PR DESCRIPTION
**Pull request by ajaggapa**
MCP update is happening ~12 seconds while deleting an MC

### CODE REVIEW

File: tests/cnf/core/network/sriov/tests/rdmametrics.go [hunk 1]
```diff
@@ -133,7 +133,7 @@ var _ = Describe("rdmaMetrics", Ordered, Label(tsparams.LabelRdmaMetricsTestCase
 				Expect(err).ToNot(HaveOccurred(), "Failed to delete rdma exclusive mode machine config")
 
 				err = netenv.WaitForMcpStable(
-					APIClient, tsparams.MCOWaitTimeout, tsparams.DefaultStableDuration, NetConfig.CnfMcpLabel)
+					APIClient, tsparams.MCOWaitTimeout, 1*time.Minute, NetConfig.CnfMcpLabel)
```
Comment (gkopels) [inline tests/cnf/core/network/sriov/tests/rdmametrics.go:136]: You have another 1*time.Minute on line 71. Maybe make a var?

Comment (ajaggapa) [reply tests/cnf/core/network/sriov/tests/rdmametrics.go:136]: Can i change below var to 1 minute or else add a new var ?

https://github.com/openshift-kni/eco-gotests/blob/44e389f89adf048805cfbccfb124e3b2d7c148cc/tests/cnf/core/network/sriov/internal/tsparams/sriovvars.go#L21

**Review by evgenLevin — APPROVED** [review]
/lgtm

**Comment by gkopels** [comment]
lgtm

**Review by gkopels — APPROVED** [review]

**Pull request merged**